contents/handbook/people/team-structure/core-experience.md

```diff
@@ -24,51 +27,48 @@ organization in matters of product design, user experience, and frontend & API t
 
 [See team structure page](/handbook/people/team-structure/team-structure)
 
+
```

Comment:
I do think we need a more concrete team mission? If you were to describe to a new team member, what does Core Experience do in 5 seconds, what would we say?

Reply:
I'd use the F-word...

Reply:
I like this as a mission: https://github.com/PostHog/posthog.com/pull/2780#discussion_r787981843